bases/world_oecd_public_finance/country_data/table_config.yaml

```diff
@@ -0,0 +1,2591 @@
+      bigquery_type: float64
+      description: Corporate Income Taxes
+      temporal_coverage:
+          - 1965(1)2016
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: revenue_environmental_tax
+      bigquery_type: float64
+      description: Environmental Taxes
+      temporal_coverage:
+          - 1994(1)2015
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: revenue_other_consumption_tax
+      bigquery_type: float64
+      description: Other Consumption Taxes
+      temporal_coverage:
+          - 1994(1)2015
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: revenue_immovable_property_tax
+      bigquery_type: float64
+      description: Taxes on Immovable Property
+      temporal_coverage:
+          - 1965(1)2016
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: revenue_other_property_tax
+      bigquery_type: float64
+      description: Other Property Taxes
+      temporal_coverage:
+          - 1965(1)2016
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: revenue_sales_goods_services_tax
+      bigquery_type: float64
+      description: Sales of Goods & Services
+      temporal_coverage:
+          - 1959(1)2017
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: revenue_other_non_property_tax
+      bigquery_type: float64
+      description: Other Nonproperty Taxes
+      temporal_coverage:
+          - 1994(1)2015
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging:
+      is_partition:
+    - name: revenue_property_income
+      bigquery_type: float64
+      description: Property Income
+      temporal_coverage:
+          - 1960(1)2019
```

Comment:
aqui pode ficar:
```
- 1960(1)

Reply:
o comentário se aplica às demais variáveis que terminam em 2019, revisar as outras tbm

Reply:
isso mesmo, mas só no caso de começar em 1945 e o `ano_final` for diferente da cobertura temporal da tabela